packages/ui/src/app/components/OnboardingOverlay/OnBoardingOverlay.tsx

```diff
@@ -79,46 +79,57 @@ export const OnBoardingOverlay = () => {
   const steps = asOnBoardingSteps(onBoardingSteps, status)
 
   return (
-    <MainWrapper>
-      <Wrapper>
-        <TextContainer>
-          <TextHuge bold>Become a member</TextHuge>
-          <TextSmall onClick={toggle}>Show how {!isOpen ? <ArrowDownIcon /> : <ArrowUpExpandedIcon />}</TextSmall>
-        </TextContainer>
-        <StepperContainer>
-          <HorizontalStepper steps={steps} />
-        </StepperContainer>
-        <ButtonContainer>
-          <ButtonPrimary size="large" onClick={openOnBoardingModal}>
-            Join now
-          </ButtonPrimary>
-        </ButtonContainer>
-      </Wrapper>
-      <StyledDropDown isDropped={isOpen}>
-        <DropdownContent>
-          <DrawerContainer title="What are the benefits?">
-            <BenefitsTable />
-          </DrawerContainer>
-          <DrawerContainer title="How to become a member?">
-            <VerticalStaticStepper steps={innerStaticStepperSteps} />
-            <ButtonPrimary onClick={openOnBoardingModal} size="large">
-              Continue
+    <>
+      <TopSpace />
+      <MainWrapper>
+        <Wrapper>
+          <TextContainer>
+            <TextHuge bold>Become a member</TextHuge>
+            <TextSmall onClick={toggle}>Show how {!isOpen ? <ArrowDownIcon /> : <ArrowUpExpandedIcon />}</TextSmall>
+          </TextContainer>
+          <StepperContainer>
+            <HorizontalStepper steps={steps} />
+          </StepperContainer>
+          <ButtonContainer>
+            <ButtonPrimary size="large" onClick={openOnBoardingModal}>
+              Join now
             </ButtonPrimary>
-          </DrawerContainer>
-        </DropdownContent>
-      </StyledDropDown>
-    </MainWrapper>
+          </ButtonContainer>
+        </Wrapper>
+        <StyledDropDown isDropped={isOpen}>
+          <DropdownContent>
+            <DrawerContainer title="What are the benefits?">
+              <BenefitsTable />
+            </DrawerContainer>
+            <DrawerContainer title="How to become a member?">
+              <VerticalStaticStepper steps={innerStaticStepperSteps} />
+              <ButtonPrimary onClick={openOnBoardingModal} size="large">
+                Continue
+              </ButtonPrimary>
+            </DrawerContainer>
+          </DropdownContent>
+        </StyledDropDown>
+      </MainWrapper>
+    </>
   )
 }
 
+const TopSpace = styled.div`
+  width: 100%;
+  height: 90px;
+`
+
 const MainWrapper = styled.div`
-  position: relative;
+  position: fixed;
+  top: 0;
+  width: 100%;
```

Comment:
Hi @talentdev0807, First of all thank you for this PR. Just one small fix has to be added - for now <ButtonPrimary>Join now </ButtonPrimary> isn’t visible. Two quick solution is to change width for a main wrapper or set up padding on Join now button